astrodendro/io/util.py

```diff
@@ -140,13 +140,19 @@ def _fast_reader(index_map, data):
 
     # ndimage ignores 0 and -1, but we want index 0
     object_slices = ndimage.find_objects(index_map+1)
+
+    # find_objects returns a tuple that includes many None values that we
+    # need to get rid of.
+    object_slices = [x for x in object_slices if x is not None]
+
```

Comment:
`git blame someoneelse` ?

I have little to no recollection of this; I remember trying to do something to speed up dendro reading and then testing it a lot locally... but I don't think I touched the tests...?